docs/getting-started/screenshots.md

```diff
@@ -1,4 +1,81 @@
-# fastlane screenshots
+Capturing and Processing App Screenshots with fastlane
```

Comment:
Hm, I'm fine with either way, but my suggestion would be to reserve backticks for when we want to show a command that needs to be run. If we just want to talk about the name of a tool, I'd suggest that we maybe just **bold** it instead?

So:
"
To use **snapshot** run `fastlane snapshot`
"